kubernetes.js

```diff
@@ -285,7 +326,10 @@ const createService = async (project, options) => {
 const createIngress = async (project, options) => {
     const prefix = project.safeName.match(/^[0-9]/) ? 'srv-' : ''
 
+    console.log("K8S DRIVER: start parse ingress template");
```

Comment:
All the  `console.log()` calls need removing or changing to a suitable `this._app.log.*` method